app/controllers/login.js

```diff
@@ -1,5 +1,7 @@
-import Ember from 'ember';
+import { Controller, computed } from 'ember';
 
 import OsfTokenLoginControllerMixin from 'ember-osf/mixins/osf-token-login-controller';
 
-export default Ember.Controller.extend(OsfTokenLoginControllerMixin);
+export default Controller.extend(OsfTokenLoginControllerMixin, {
+    login: computed.alias('model'),
```

Comment:
I'm good with that. I'll go ahead and turn it off in the COS config and then update this PR with after reverting this change.